### ISSUE TITLE
[Addon] Ships don't pass borders

### ISSUE THREAD

**Issue by Spikeone** [Return-To-The-Roots/s25client#184]
An addon which would permit border passing by civil ships could also be great. It's importat that team members should be able to pass their allys borders without any problems.
Also this should not affect any exploration expedition or military company.
This addon could prevent some "bug" alike mechanics, sometimes civil transport ships pass enemy harbors spotting them for an attack that way some island can sometimes not be occupied, this could also be fixed by mapdesign but in most cases, that wont happen anymore sadly.

**Comment by Flow86** [comment]
well, I think thats more a feature than a bug currently. but optional we could set that "avoid enemy territory" (not restrict - since sometimes you can't go to a harbor without touching enemy territory)

**Comment by Spikeone** [comment]
Yeah, I also consider this as an addon - thats why I didn't label it wish, and as I also said those bug alike mechanics could be prevented by mapdesign (but no "new" editor would be aware). I just think this could be a really good feature also for mappers as for example you can add an island, if a player controls this island some areas for other players could be blocked so they have to attack those first.
